packer/install-dependencies.sh

```diff
@@ -307,25 +307,15 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.0.30
 1.0.29
-1.0.28
 "
 for AZURE_NPM_VERSION in ${AZURE_NPM_VERSIONS}; do
     CONTAINER_IMAGE="${AZURE_CNIIMAGEBASE}/azure-npm:v${AZURE_NPM_VERSION}"
     pullContainerImage "docker" ${CONTAINER_IMAGE}
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
-AZURE_VNET_TELEMETRY_VERSIONS="
```

Comment:
Do 1.0.29 and 1.0.28 work without the sidecar? If not, let's keep this in the VHD spec as well.

Reply:
No, they won't work w/o sidecar. Should I revert the changes in parts/k8s/containeraddons/kubernetesmasteraddons-azure-npm-daemonset.yaml as well?

Reply:
Got it. We can add a conditional go template expression to the yaml spec "if < 1.0.30 then include these extra pods" thing.

Let's not worry about that now, we can iterate there. In the meantime for convenience let's keep everything in the VHD that was already there.